templates/2025/home/all_years.html

```diff
@@ -87,26 +87,26 @@ <h3 class="h3 g-font-size-15 g-color-white mb-0">Online</h3>
-                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2024/img/11.jpg' %});"></div>
-                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2024/img/13.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/1.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/2.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/3.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/4.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/5.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/6.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/7.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/8.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/9.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/10.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/11.jpg' %});"></div>
+                <div class="js-slide g-bg-img-hero g-min-height-50vh" style="background-image: url({% static '2025/img/13.jpg' %});"></div>
               </div>
             </div>
 
             <div class="col-md-6 d-flex g-min-height-50vh g-theme-color-gray-dark-v1 g-px-50 g-py-80 g-py-50--md g-py-20--lg">
               <div class="align-self-center w-100">
                 <h2 class="text-uppercase g-font-weight-700 g-font-size-20 g-color-white g-mb-10">PyCon Uganda will be Online</h2>
-                <h3 class="text-uppercase g-font-weight-500 g-font-size-13 g-color-white g-mb-20">PyCon Uganda is the gathering of the Pan African Python Community.</h3>
-                <p class="g-font-size-14 g-color-white ">We had the very first edition in 2019 and while the conference cannot take place in person this year, we would still be having it online making it PycOnline Africa 2020!</p>
+                <h3 class="text-uppercase g-font-weight-500 g-font-size-13 g-color-white g-mb-20">PyCon Uganda is the gathering of the Pan Ugandan Python Community.</h3>
```

Comment:
there is no `Pan Ugandan Python Community` maybe we could just say Python Community